### ISSUE TITLE
Feature Request: Suspend computer on long break

### ISSUE THREAD

**Issue by jobayersajal1** [slgobinath/SafeEyes#389]
Hi,
First of all thanks a lot for such amazing app.
I guess,when I am going for a long break like 15 minutes,my pc/laptop should get a rest on physical level too.Instead of wasting resource like displaying count time on screen,it can go to sleep mode.Most of us will eventually wont look at the screen during break.
Thanks

**Comment by slgobinath** [comment]
I think it deserves a separate plugin. Very similar to the Screensaver plugin, we need a plugin to execute a command that puts the system sleep/hibernate/shutdown.

**Comment by jobayersajal1** [comment]
Agreed, Usually there comes a lock button on top-right corner during short break. I was thinking, is it possible just to add sleep button on top right corner during  long break?
